src/Qdrant.Client/Grpc/PointsSelector.cs

```diff
@@ -8,12 +8,22 @@ public partial class PointsSelector
 	/// <summary>
 	/// Implicitly converts a <see cref="Guid" /> ID to a new instance of <see cref="PointsSelector"/>
 	/// </summary>
-	public static implicit operator PointsSelector(Guid id)
-		=> new() { Points = new PointsIdsList { Ids = { new PointId { Uuid = id.ToString() } } } };
+	public static implicit operator PointsSelector(Guid id) => new Guid[] { id };
```

Comment:
Suggest keeping this as- it's a little more obvious than chaining on to another implicit overload, and avoids the unnecessary array allocation.

Reply:
https://github.com/qdrant/qdrant-dotnet/pull/41/commits/07bea9863f22f407534d184fdc6f8991e4ffd2cf